### ISSUE TITLE
Unicode input is about JSON encoders, not about YAML.

### ISSUE THREAD

**Issue by ElectronicRU** [cblp/yaml-sucks#13]
The last row of the table is misleading -- JSON spec has escape sequences, so the string is the same in all cells, but is expressed differently in JSON.

**Comment by champignoom** [comment]
Indeed. Adding `ensure_ascii=False` to `json.dumps()` should eliminates the difference.

**Comment by ssokolow** [comment]
> The primary objective of this revision is to bring YAML into compliance with JSON as an official subset.
> -- https://yaml.org/spec/1.2/spec.html

(As of YAML 1.2, it's officially intended that all valid JSON can be parsed as valid YAML.)

**Comment by champignoom** [comment]
The issue should be titled "Unicode *output* is about JSON *decoders*"

**Comment by champignoom** [comment]
@ssokolow How's that relevant here?

**Comment by ssokolow** [comment]
...it's not. I underestimated how sleep-deprived I am and misread the issue.

Sorry about that.
